airflow/jobs/scheduler_job.py

```diff
@@ -497,13 +500,15 @@ def update_import_errors(session, dagbag):
                 stacktrace=stacktrace))
         session.commit()
 
+    # pylint: disable=too-many-return-statements,inconsistent-return-statements,too-many-branches
```

Comment:
Maybe it's worth adding the missing None values so that this method returns DagRun or None? This should solve `inconsistent-return-statements` error.

Reply:
Done